plugins/removeformat/plugin.js

```diff
@@ -35,6 +35,11 @@ CKEDITOR.plugins.removeformat = {
 					range;
 
 				while ( ( range = iterator.getNextRange() ) ) {
+					// Prevent of removing `span` element with `lang` and `dir` attribute (#779).
+					editor.addRemoveFormatFilter( function( element ) {
```

Comment:
Then simple condition like:
```javascript
if ( editor.addRemoveFormatFilter ) {
    // Do the magic.
}
```
should be sufficient.

If `removeFormat` plugin is removed, then this filter won't be needed.